resolver/GlobalPass.cc

```diff
@@ -327,7 +327,7 @@ InlinedVector<core::SymbolRef, 4> ParentLinearizationInformation::fullLinearizat
     return res;
 }
 
-void computeLinearization(core::GlobalState &gs) {
+void Resolver::computeLinearization(core::GlobalState &gs) {
```

Comment:
We expose this so we can do just the linearization and none of the rest of the `finalizeSymbols` pass on the fast path.